android-vjestina-homework-movie-app-data-mock/app/src/main/java/agency/five/codebase/android/movieapp/ui/component/UserScoreProgressBar.kt

```diff
@@ -0,0 +1,78 @@
+package agency.five.codebase.android.movieapp.ui.component
+
+import agency.five.codebase.android.movieapp.R
+import agency.five.codebase.android.movieapp.ui.theme.Typography
+import agency.five.codebase.android.movieapp.ui.theme.spacing
+import androidx.compose.animation.core.animateFloatAsState
+import androidx.compose.animation.core.tween
+import androidx.compose.foundation.Canvas
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.StrokeCap
+import androidx.compose.ui.graphics.drawscope.Stroke
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Dp
+
+private const val DEGREES_IN_CIRCLE = 360
+private const val PERCENTAGE_FACTOR = 10f
+private const val STARTING_POSITION_OF_CIRCLE = -90f
+private const val ANIMATION_DURATION = 1000
+
+@Composable
+fun UserScoreProgressBar(
+    modifier: Modifier = Modifier,
+    score: Float,
+    radius: Dp = dimensionResource(id = R.dimen.progress_circle_radius),
```

Comment:
Radius should be calculated from size of the composable (passed with modifier) and not the other way around. 
Remove this parameter and let the size be set from the calling site -> remove .size(radius * 2f) from Box and use `Modifier.fillMaxSize()` extension function in Canvas.